src/backend/distributed/citus--8.0-3--8.0-4.sql

```diff
@@ -0,0 +1,11 @@
+/* citus--8.0-3--8.0-4 */
+SET search_path = 'pg_catalog';
+
+CREATE OR REPLACE FUNCTION lock_relation_if_exists(table_name text, lock_mode text)
```

Comment:
I think using text is probably a good idea. We've previously used integers in the UDFs for shard metadata, but this will break rolling upgrades if the lock mode numbering ever changes. Do you think we should change those functions as well? (or at least, open an issue / check if PG11 breaks)

Reply:
Makes sense, @mtuncer will manually check the lockmodes for pg11.  #2372 is opened for reference.

Reply:
verified lock mode defines are not changed in PG11